### PR TITLE
ci: push to hub on merge

### PR DESCRIPTION
**Pull request by cristianmtr**
Part of https://github.com/jina-ai/jina/issues/2884

### CODE REVIEW

**Review by Yongxuanzhang — DISMISSED** [review]

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - id: push
+        run: ./scripts/push.sh
+        env:
+          secret: ${{ secrets.HUBBLE_SECRET }}
+          id: ${{ secrets.HUBBLE_ID }}
```
Comment (nan-wang) [inline .github/workflows/cd.yml]: Keep the naming consistent with hubble. 

```suggestion
          id: ${{ secrets.HUBBLE_UUID }}
```

File: scripts/push.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+set -ex
+
+pip install jina[hub]~=2.0
```
Comment (nan-wang) [inline scripts/push.sh]: I suggest using the latest master by default and make the Jina version an environment variable.

```suggestion
pip install jina[hub]~=2.0
```

**Review by nan-wang — DISMISSED** [review]

File: .github/workflows/cd.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  pull_request:
+    branches:
+      - master
+    types: [ closed ]
+
+
+jobs:
+  push-hub:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
```
Comment (mapleeit) [inline .github/workflows/cd.yml:14]:
```suggestion
    runs-on: ubuntu-latest
    container: 
      image: jinaai/jina:master
      options: "--entrypoint /bin/bash"
```

File: .github/workflows/cd.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - id: push
+        run: ./scripts/push.sh
```
Comment (mapleeit) [inline .github/workflows/cd.yml]:
```suggestion
        run: jina hub push --force ${{ secrets.HUBBLE_ID }} --secret ${{ secrets.HUBBLE_SECRET }} .
```

**Review by mapleeit — CHANGES_REQUESTED** [review]
Removing `./scripts/push.sh` and using container with directly run would be more clean and clear from my point of view.

File: scripts/push.sh (hunk 2 above)
Comment (JoanFM) [reply scripts/push.sh]: Ideally this should map to `jina` version of the executor itself

Comment (cristianmtr) [reply scripts/push.sh]: This could be done with using dependabot, I think

Comment (cristianmtr) [reply scripts/push.sh]: Diff PR though, once we decide on versioning

**Comment by cristianmtr** [comment]
@JoanFM @nan-wang @mapleeit Re-check

**Review by mapleeit — APPROVED** [review]
LGTM

**Pull request merged**